services/checks/checks.go

```diff
@@ -927,6 +1046,18 @@ func (s *Service) collectChecks(ctx context.Context) {
 	}
 
 	checks = s.filterSupportedChecks(checks)
+	for i, c := range checks {
+		cs, err := models.FindCheckStateByName(s.db.Querier, c.Name)
+		if err != nil {
+			// we use warn because this might not always be a failure, its possible
+			// that there were no interval changes to be applied in the first place.
+			s.l.Warnf("Unable to re-apply interval changes to check: %s : %+v", c.Name, err)
```

Comment:
maybe then there is no reason to warn at all?

Reply:
removed

Reply:
forgot to remove this, will do in a few mins

Reply:
actually I thought better than removing it, is to log it if we receive an error other than `reform.ErrNoRows` from the DB

let me know what you think